templates/bot_status.j2

```diff
@@ -1,11 +1,15 @@
-### Components
+<details><summary><b><u>Components</u></b></summary>
```

Comment:
So I've tested this in a preview mode and it looks like `<u>` has no effect. I'd remove it to avoid confusion.
```suggestion
<details><summary><b>Components</b></summary>
```